packages/experimental/RadioGroup/src/RadioGroup/useRadioGroup.ts

```diff
@@ -31,13 +31,12 @@ export const useRadioGroup = (props: RadioGroupProps): RadioGroupInfo => {
   return {
     props: {
       ...props,
-      disabled,
       required,
-      accessible: true,
       accessibilityRole: 'radiogroup',
       accessibilityLabel: accessibilityLabel ?? label,
       accessibilityState: getAccessibilityState(state.disabled, state.required, accessibilityState),
       defaultTabbableElement: selectedButtonRef,
+      focusable: !state.disabled,
```

Comment:
Is the radiogroup supposed to be focusable separately from radios?

Also this change doesn't seem to be related to the PR, it's easy to miss this change in a large PR like this, so in the future I'd recommend making separate PRs for changes so that they're reviewed properly

Reply:
+1 I don't think the RadioGroup should be focusable itself, just the Radios